### PR TITLE
[Ewerton] - sage - aprenda sobre oraculos

### PR DESCRIPTION
**Pull request by guizostudios**
### Thanks for submitting this tutorial for Celo Sage

Before you hit "Create Pull Request", make sure you have read the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines) and reviewed the [Sample Tutorial Structure](https://docs.celo.org/community/celo-sage/tutorial-structure).

Please confirm the following before submitting your tutorial for review -

- [X] My tutorial follows the [Tutorial Contribution Guidelines](https://docs.celo.org/community/celo-sage/contribution-guidelines).
- [X] The title of the Pull Request matches the title of my tutorial.
- [X] I've followed the tutorial template structure.
- [X] I've proofread the tutorial for spelling and grammar mistakes.
- [X] I've thoroughly explained the code, with inline code comments and text explanations.
- [X] I've run a linter & prettifier to format my code blocks.
- [X] I've run the tutorial to test it and the code works.

> NOTE: We cannot review your Pull Request if it does not meet the standard outlined above.
> Reviewers - @joenyzio & @viral-sangani

### CODE REVIEW

File: blog/03-27-2023-aprenda-sobre-oraculos/aprenda_sobre_oraculos.md [hunk 1]
```diff
@@ -0,0 +1,57 @@
...
+authors:
+- name: Ewerton Lopes
+  title: Founder of Guizo Studios
+  url: https://github.com/guizostudios
+  image_url: https://avatars.githubusercontent.com/u/97633823
+tags: [celo, solidity, developers, remix, celosage]
+hide_table_of_contents: true
+slug: /tutorials/aprenda-sobre-oraculos
+---
+
```
Comment (Ishanpathak1) [inline blog/03-27-2023-aprenda-sobre-oraculos/aprenda_sobre_oraculos.md:13]: Add this line here 
import YouTube from '@components/YouTube';

File: blog/03-27-2023-aprenda-sobre-oraculos/aprenda_sobre_oraculos.md [hunk 2]
```diff
@@ -0,0 +1,57 @@
...
+Para este tutorial não é necessário conhecimento prévio. As ferramentas utilizadas serão:
+
+- Remix: [Remix](https://remix.ethereum.org/) é uma Integrated Development Environment (IDE) para desenvolvimento de smart contracts na blockchain Ethereum. Ele fornece uma plataforma fácil de usar para escrever, testar e depurar smart contracts escritos na linguagem Solidity. Além disso, a IDE Remix também fornece ferramentas para simular e executar smart contracts em diferentes ambientes de rede, incluindo a rede principal Ethereum e redes de teste. Ele é uma ferramenta popular entre desenvolvedores de smart contracts, pois oferece recursos avançados para facilitar o desenvolvimento e teste de contratos inteligentes.
+
+## Tutorial
+
+Confira no video como construir um smart contract utilizando oraculos em Celo.
+
+[![Aprenda sobre oraculos](../../src/data-tutorials/showcase/intermediate/aprenda_sobre_oraculos.png)](https://youtu.be/1YQqLxm1SqE)
```
Comment (Ishanpathak1) [inline blog/03-27-2023-aprenda-sobre-oraculos/aprenda_sobre_oraculos.md:37]: and replace this with
<YouTube videoId="1YQqLxm1SqE"/>

**Review by Ishanpathak1 — COMMENTED** [review]
Make the changes if you could

File: blog/03-27-2023-aprenda-sobre-oraculos/aprenda_sobre_oraculos.md (hunk 2 above)
Comment (guizostudios) [reply blog/03-27-2023-aprenda-sobre-oraculos/aprenda_sobre_oraculos.md:37]: Thanks! I'll do that in the next videos

**Review by developerkunal — APPROVED** [review]
Approved @Ishanpathak1

**Pull request merged**